Obsidian/Commands/MainCommandModule.cs

```diff
@@ -7,6 +7,7 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
+using System.Net.Mime;
```

Comment:
Unnecessary using?

Reply:
I don't know when and why I added it, but yeah. It will be removed in the next pull.